prototype_source/pt2e_quant_ptq_static_x86_inductor.rst

```diff
@@ -0,0 +1,188 @@
+quantize the model.
+
+::
+
+    quantizer = X86InductorQuantizer()
+    quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())
+
+.. note::
+
+- The default quantization configuration in ``X86InductorQuantizer`` uses 8-bits for both activations and weights.
+  When Vector Neural Network Instruction is not available, the onednn backend silently chooses kernels that assume
+  `multiplications are 7-bit x 8-bit <https://oneapi-src.github.io/oneDNN/dev_guide_int8_computations.html#inputs-of-mixed-type-u8-and-s8>`_. In other words, potential
+  numeric saturation and accuracy issue may happen when running on CPU without Vector Neural Network Instruction.
+
+After we import the backend-specific Quantizer, we will prepare the model for post-training static quantization.
+``prepare_pt2e`` folds BatchNorm operators into preceding Conv2d operators, and inserts observers in appropriate places in the model.
+
+::
+
+    prepared_model = prepare_pt2e(exported_model, quantizer)
+
+Now, we will calibrate the ``prepared_model`` after the observers are inserted in the model.
+
+::
+
+    # We use the dummy data as an example here
+    prepared_model(*example_inputs)
+
+    # Alternatively: user can define the dataset to calibrate
+    # def calibrate(model, data_loader):
+    #     model.eval()
+    #     with torch.no_grad():
+    #         for image, target in data_loader:
+    #             model(image)
+    # calibrate(prepared_model, data_loader_test)  # run calibration on sample data
+
+Finally, we will convert the calibrated Model to a quantized Model. ``convert_pt2e`` takes a calibrated model and produces a reference quantized model.
+
+::
+
+    converted_model = convert_pt2e(prepared_model)
+
+After these steps, we finished running the quantization flow and we will get the reference quantized model.
+
+
+3. Lower into Inductor
+------------------------
+
+After we get the reference quantized model, we will further lower it to the inductor backend.
+
+::
+
+    optimized_model = torch.compile(converted_model)
+
+    # Running some benchmark
+    optimized_model(*example_inputs)
+
+
+Put all these codes together, we will have the toy example code.
+Please note that since the Inductor ``freeze`` feature does not turn on by default yet, run your example code with ``TORCHINDUCTOR_FREEZING=1``.
+For example:
+
+::
+
+    TORCHINDUCTOR_FREEZING=1 python example_x86inductorquantizer_pytorch_2_1.py
+
+4. Conclusion
+---------------
+
+With this tutorial, we introduce how to use Inductor with X86 CPU in PyTorch 2.0 Quantization. Users can learn about
+how to use ``X86InductorQuantizer`` to quantize a model and lower it into the inductor with X86 CPU devices. If you need new quantization
+features for the ``X86InductorQuantizer``, please submit a new issue in PyTorch GitHub repository labeled as ``oncall: quantization`` and assign
+it to `Leslie Fang <https://github.com/leslie-fang-intel>`_.
```

Comment:
```suggestion
how to use ``X86InductorQuantizer`` to quantize a model and lower it into the inductor with X86 CPU devices. 
```
I don't think we need to mention this. There is a triage process in the repo and the proper oncall will be tagged as needed.